### ISSUE TITLE
[Suggestion] Config for Oil Lakes

### ISSUE THREAD

**Issue by aking4125** [GregTechCE/GregTech#1257]
The Oil Spheres are configurable by default in the worldgen folders via their respective .json files already generated. But that doesn't affect the standard Oil Lakes from appearing in pretty much any dimension possible.

Perhaps a config option to either turn them off or create a dimension blacklist/whitelist?

Rather minor suggestion, however.

**Comment by LAGIdiot** [comment]
Thank you for creating this issue.

But I think you are mistaken. First thing is all our worldgen is configurable which means removing our *.json files will disable it. Second is we don't have "lake" generator configuration so it is not possible for us to generate them.

I guess that you have other mod which has Oil (but thanks to Forge and fluid "standardization" it is replace by our Oil) and generates them. Or you have worldgen mod which has some kind of integration (probably just looking for Oil fluid) and generates it.

Please let us know when you find who cause it if you find it.

**Comment by aking4125** [comment]
Another mod creating the lakes....Hmmm

The reason why I suspected it to be Gregtech was the oil within the lakes is gregtech. Specifically the light type but only the light type. But since you mentioned a forge & fluid "Standardization". It's possible it's replacing it and Gregtech is overriding it.

I'll take a deeper look. I did find it suspicious that nearly all worldgen was configurable in the .jsons except for the lakes.
